clone/naverclone.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="utf-8" />
+  <title>NAVER</title>
+  <link rel="shortcut icon" type="image/x-icon" href="./favicon.ico" />
+  <link rel="stylesheet" href="./naverclone.css" />
+</head>
+
+<body>
+  <header>
+    <div id="header-banner">
+      <div id="header-movenmentsite">
+        <a>네이버를 시작페이지로</a>
+        <a>쥬니버네이버</a>
+        <a>해피빈</a>
+      </div>
+      <h1>
+        <a href="./naverclone.html">
+          <span class="blind">네이버</span>
+        </a>
+      </h1>
+      <div id="header-search">
+        <fieldset>
+          <legend class="blind">검색</legend>
+          <input type="text" />
+          <button>
+            <span class="blind">검색</span>
+            <span class="ico_search"></span>
```

Comment:
```suggestion
            <span class="ico-search"></span>
```

`-`를 사용하는게 컨벤션입니당
다른 부분도 많이 보이네요. 같이 수정해주세요!